client/src/components/Projects/ColumnHeaderPopups/StatusPopup.js

```diff
@@ -72,7 +118,10 @@ const StatusPopup = ({
         />
         <hr style={{ width: "100%" }} />
       </div>
-      <div>(Under Construction)</div>
+      <UniversalSelect
```

Comment:
You need two separate drop-downs on this popup. One controls criteria.status, which should have possible values ["active", "deleted", or "all"] the second controls criteria.type, which should have possible values of ["draft", "snapshot" or "all"]